.github/workflows/nativeBuild.yml

```diff
@@ -157,6 +157,7 @@ jobs:
           version: '22.3.0'
           java-version: '11'
           components: 'native-image'
+          native-image-musl: 'true'
```

Comment:
Before using musl, running native on WSL resulted below error
```
[Error - 10:32:54 AM] Connection to server is erroring. Shutting down server.
[Trace - 10:32:54 AM] Sending request 'shutdown - (1)'.
./server-linux: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by ./server-linux)
./server-linux: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by ./server-linux)
```

After using the musl, we see an graalvm error. But this does prove that, musl libc is used in this build

```
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [resource:/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [resource:/org/slf4j/impl/StaticLoggerBinder.class#1]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [ch.qos.logback.classic.util.ContextSelectorStaticBinder]
Exception in thread "main" java.util.concurrent.ExecutionException: com.oracle.svm.core.jdk.UnsupportedFeatureError: Runtime reflection is not supported for public org.eclipse.lsp4j.WorkspaceEditChangeAnnotationSupportCapabilities()

```